src/components/CreateRules/UploadRules.vue

```diff
@@ -227,6 +264,32 @@ export default class UploadRules extends Mixins(DocumentMixin) {
   /** Called when component is created. */
   private created (): void {
     this.uploadRulesDoc = this.getCreateRulesStep.rulesDoc as File
+    this.rulesConfirmed = this.getCreateRulesStep.rulesConfirmed
+  }
+
+  @Watch('getShowErrors')
+  private onShowErrorsChanged (): void {
+    if (this.getShowErrors && this.$refs.confirmRulesChk) {
+      this.$refs.confirmRulesChk.validate()
+    }
+  }
+
+  @Watch('$route')
+  private async scrollToInvalidComponent (): Promise<void> {
```

Comment:
I have a concern about watching the route here: I believe this component is always rendered (but maybe not shown), so it's watching the route even when the user is not on this page. Would this event handler cause issues on other pages?

Normally I would expect a route watcher to be in a high level component (App?) so that it applies over all pages.

Reply:
i plan on picking up the memorandum UI ticket.  will take a look at this when i work it if that's alright?

Reply:
Sounds good.